Doc/includes/noddy.c

```diff
@@ -3,70 +3,37 @@
 typedef struct {
     PyObject_HEAD
     /* Type-specific fields go here. */
-} noddy_NoddyObject;
+} NoddyObject;
 
-static PyTypeObject noddy_NoddyType = {
+static PyTypeObject NoddyType = {
     PyVarObject_HEAD_INIT(NULL, 0)
-    "noddy.Noddy",             /* tp_name */
-    sizeof(noddy_NoddyObject), /* tp_basicsize */
-    0,                         /* tp_itemsize */
-    0,                         /* tp_dealloc */
-    0,                         /* tp_print */
-    0,                         /* tp_getattr */
-    0,                         /* tp_setattr */
-    0,                         /* tp_reserved */
-    0,                         /* tp_repr */
-    0,                         /* tp_as_number */
-    0,                         /* tp_as_sequence */
-    0,                         /* tp_as_mapping */
-    0,                         /* tp_hash  */
-    0,                         /* tp_call */
-    0,                         /* tp_str */
-    0,                         /* tp_getattro */
-    0,                         /* tp_setattro */
-    0,                         /* tp_as_buffer */
-    Py_TPFLAGS_DEFAULT,        /* tp_flags */
-    "Noddy objects",           /* tp_doc */
-    0,                         /* tp_traverse */
-    0,                         /* tp_clear */
-    0,                         /* tp_richcompare */
-    0,                         /* tp_weaklistoffset */
-    0,                         /* tp_iter */
-    0,                         /* tp_iternext */
-    0,                         /* tp_methods */
-    0,                         /* tp_members */
-    0,                         /* tp_getset */
-    0,                         /* tp_base */
-    0,                         /* tp_dict */
-    0,                         /* tp_descr_get */
-    0,                         /* tp_descr_set */
-    0,                         /* tp_dictoffset */
-    0,                         /* tp_init */
-    0,                         /* tp_alloc */
-    PyType_GenericNew,         /* tp_new */
+    .tp_name = "noddy.Noddy",
+    .tp_doc = "Noddy objects",
+    .tp_basicsize = sizeof(NoddyObject),
+    .tp_itemsize = 0,
+    .tp_flags = Py_TPFLAGS_DEFAULT,
+    .tp_new = PyType_GenericNew,
 };
 
 static PyModuleDef noddymodule = {
     PyModuleDef_HEAD_INIT,
-    "noddy",
-    "Example module that creates an extension type.",
-    -1,
-    NULL, NULL, NULL, NULL, NULL
+    .m_name = "noddy",
+    .m_doc = "Example module that creates an extension type.",
+    .m_size = -1,
 };
 
 PyMODINIT_FUNC
 PyInit_noddy(void)
 {
-    PyObject* m;
-
-    if (PyType_Ready(&noddy_NoddyType) < 0)
+    PyObject *m;
+    if (PyType_Ready(&NoddyType) < 0)
         return NULL;
 
     m = PyModule_Create(&noddymodule);
     if (m == NULL)
         return NULL;
 
-    Py_INCREF(&noddy_NoddyType);
-    PyModule_AddObject(m, "Noddy", (PyObject *)&noddy_NoddyType);
+    Py_INCREF(&NoddyType);
+    PyModule_AddObject(m, "Noddy", (PyObject *) &NoddyType);
```

Comment:
Don't know if this is covered by PEP 7, but I think that it is more common to not add a space between closing `)` and the casted value.

Reply:
Hmm, I'd think the contrary.